src/test/rpc/NoRipple_test.cpp

```diff
@@ -222,42 +226,60 @@ class NoRipple_test : public beast::unit_test::suite
 
         env(trust(gw, USD(100), alice, 0));
         env(trust(gw, USD(100), bob, 0));
+        Json::Value params;
+        params[jss::api_version] = apiVersion;
 
+        if (params[jss::api_version] < 2)
```

Comment:
There's a bunch more overhead to look up `params[jss::api_version]` and compare it to `2` than there would be to just compare `apiVersion < 2u`.

But more importantly, this check makes no sense. It makes more sense to run every test, then compare the results based on version. Yes, that will sometimes lead to repetition, but what happens if version 1 is eventually deprecated? Do you want these tests to be deleted? Of course not. You also want to run that version-2-only test under version 1 to demonstrate that the `"asdf"` is accepted as a valid value.